pyfftw/interfaces/scipy_fftpack.py

```diff
@@ -412,9 +437,41 @@ def irfft(x, n=None, axis=-1, overwrite_x=False,
                            auto_contiguous)
 
 
+# why map None:None, and not None:"forward"?
+# if I change it to map None:"forward" some `scipy_interface` tests fail.
+# see also the if-clauses in the sine/cosine transforms in `scipy_fft.py`;
+# there's some mishandling of norm=None somewhere.
+_swap_norm_dictionary = {"backward": "forward", None: None,
```

Comment:
that mapping of `None:None` was like that in the code, and I'm not sure why it shouldn't map `None:"forward"`; I've left it as is for now because some of the tests were failing otherwise (see code comments)